### PR TITLE
fix: don't assume we can get a quake of arrows

### PR DESCRIPTION
**Pull request by midgleyc**
Having an Obtuse Angel but not a quake of arrows would lead to an abort of "unable to satisfy famspec". Try to work around that.

It costs ~50k in the mall and can rise substantially, so it can breach your auto-buy price limit.

### CODE REVIEW

**Comment by Rinn** [comment]
maybe it should just prefer reanimator and then only set the quake of arrows as the famequip if you have it, because getting 2 copies is better than none.

File: packages/garbo/src/fights.ts [hunk 1]
```diff
@@ -525,9 +524,17 @@ export function dailyFights(): void {
...
 
-        const bestCopier = $familiars`Obtuse Angel, Reanimated Reanimator`.find(
-          have,
-        );
+        // use obtuse angel if have + have quake of arrows, otherwise reanimator
+        // quake of arrows is PvP-stealable and costs ~50k, so don't assume we have it
+        let bestCopier: Familiar | undefined;
+        if (
+          have($familiar`Obtuse Angel`) &&
+          retrieveItem($item`quake of arrows`)
```
Comment (s-k-z) [inline packages/garbo/src/fights.ts:532]: I don't think this is the best solution because `retrieveItem` may unequip the quake of arrows unnecessarily.

Comment (midgleyc) [reply packages/garbo/src/fights.ts]: I think this should sort it?

Comment (s-k-z) [reply packages/garbo/src/fights.ts]: Yeah!

**Review by s-k-z — APPROVED** [review]

**Pull request merged**